<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>772</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>891</width>
      <height>751</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
background-color:white;

}</string>
    </property>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>851</width>
       <height>2</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#line{
	height:1px;
	background-color: #e8e8e8;
	border:none;
}</string>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="nutriFit">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>20</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	font:600 25px &quot;Epilogue&quot;;
	color:#1D1D1D;
	background-color:transparent;
}</string>
     </property>
     <property name="text">
      <string>NutriFit</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>80</y>
       <width>851</width>
       <height>621</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
background-color:#F9F9F9;
border-radius:10px;
border:1px solid#c0c0c0
}</string>
     </property>
     <widget class="QFrame" name="cam_frame">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>811</width>
        <height>321</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
background-color: transparent;
border:1px solid #c0c0c0;
border-radius:10px
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="search_btn">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>350</y>
        <width>381</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color:#1d1d1d;
border-radius:25px;
font:600 15px &quot;Epilogue&quot;;
}</string>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>410</y>
        <width>811</width>
        <height>191</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget{
background-color:white;
color:#c0c0c0;
font:400 15px &quot;Epilogue&quot;;
border:1px solid #c0c0c0;
border-radius:10px;
padding-top: 20px;
padding-right:20px;
padding-left:20px
}</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>894</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
